src/components/common/ListPeopleAndRoles.vue

```diff
@@ -66,10 +66,12 @@
           </v-col>
 
           <v-col class="delivery-address-column">
-            <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)">
-              Same as Mailing Address
-            </p>
-            <DeliveryAddress v-else :address="orgPerson.deliveryAddress" />
+            <template v-if="isDirector(orgPerson) || isProprietor(orgPerson)">
+              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)">
+                Same as Mailing Address
+              </p>
+              <DeliveryAddress v-else :address="orgPerson.deliveryAddress" />
+            </template>
```

Comment:
Without this, sometimes it said "same as".

Now:

![image](https://user-images.githubusercontent.com/10002889/162334120-299a49ce-7e0c-405f-9ecf-95aff5329146.png)

Reply:
If only there was something to put in that empty column 🤔

Reply:
Agreed. I sent Yui this screenshot to think about whether he wants anything in there (which can be changed in a future ticket/PR).

Reply:
Yui says, "I think it’s fine for CP’s delivery address to be blank since the table needs to support the other roles that have delivery address."